burn-import/src/onnx/dim_inference.rs

```diff
@@ -297,6 +298,23 @@ fn flatten_update_outputs(node: &mut Node) {
     });
 }
 
+/// Infers the shape of a Conv1d node and replaces the shape of the output tensor.
+///
+/// The shape of the output tensor is calculated by running the actual convolution operation.
```

Comment:
The comment is outdated and applicable anymore. If you see it on Conv2d, please remove it as well.

Before a big refactor, we used to calculate the output shape by calling the Conv2d code. But we have figured out we do not need to know the shapes. But instead only a dimension of the output is needed. So this comment was forgotten to be cleaned (and probably confusing others now =D)

Reply:
Done, I removed it on both functions!